tests/core/app.test.ts

```diff
@@ -134,19 +134,19 @@ describe('Testing App routing', () => {
     )
     ;(await makeFetch(app.handler)('/abcdef')).expect(404)
   })
-  // it('"*" should catch all undefined routes', async () => {
-  //   const app = new App()
-
-  //   const server = app.handler
-
-  //   app
-  //     .get('/route', (_req, res) => void res.send('A different route'))
-  //     .all('*', (_req, res) => void res.send('Hello world'))
+  it('"*" should catch all undefined routes', async () => {
+    const app = new App()
 
-  //   await makeFetch(server)('/route').expect(200, 'A different route')
+    const server = app.handler
```

Comment:
app handler is not a server, I think it's okay to just pass the app.handler to makeFetch directly